cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -83,15 +86,15 @@ def from_domain_expert_metadata(
         prefix: str | None = None,
         namespace: Namespace | None = None,
         version: str | None = None,
-        contributor: str | list[str] | None = None,
+        creator: str | list[str] | None = None,
```

Comment:
We have a special type that parses a comma list. 

```suggestion
        creator: StrOrListType | None = None,
```